project4/project/ebs/static/ebs/css/add-blog.css

```diff
@@ -34,8 +49,14 @@
 	text-align:right;
 }
 .media-button{
+	vertical-align:bottom;
```

Comment:
remove vertical-align, as it is not required when the element is at absolute position.